src/components/projects/CreateProjectForm.vue

```diff
@@ -622,21 +654,53 @@ export default {
             }
         },
 
+        RAfromSexagesimal(rasexag) {
+
+          let raarray = rasexag.replace(":", " ").replace(":", " ").replace(".", " ").split(" ")
+
+          let rahrs = Number(raarray[0]) + Number(raarray[1]/60) + Number(raarray[2]/3600)
+          console.log(rahrs)
+       
+          return rahrs 
+        
+        },
+
+        DECfromSexagesimal(decsexag) {
+          
+          let decarray = decsexag.replace(":", " ").replace(":", " ").replace(".", " ").split(" ")
+
+          if (Math.sign(Number(decarray[[0]])) == -1) {            
+             return  Number(decarray[0]) - Number(decarray[1]/60) - Number(decarray[2]/3600)
+          } 
+          else {
+             return Number(decarray[0]) + Number(decarray[1]/60) + Number(decarray[2]/3600)
+          }
+
+        },
+        
+
+
```

Comment:
Not something that has to happen now, but if we expect to use these functions in multiple places, it might be a good idea to throw them in one of the utils files (like helpers.js) to avoid clutter in each vue component. (To be fair, that can be said of a lot of the already written code in other places too - just thinking out loud here and not a necessity!)